fastlane/Fastfile

```diff
@@ -234,6 +234,43 @@ lane :run_pod_install do |options|
   end
 end
 
+desc "Run automatic bump in other repos"
+lane :bump_hybrid_dependencies do |options|
+  bump = current_version_number
+  repo_name = options[:repo_name],
+
+  UI.message("Triggering pipeline to bump #{repo_name} dependency to #{bump}")
+
+  require 'net/http'
+  require 'uri'
+  require 'json'
```

Comment:
Hmm not sure if it's best practice to import these at the top of the file... But then again, these are only used here so it seems better... I guess I'm ok with this.